docs/release-notes/release-notes-0.14.2.md

```diff
@@ -24,6 +24,7 @@
   state](https://github.com/lightninglabs/lightning-terminal/pull/985) instead
   of the Revoked state if it is in fact being revoked due to the session expiry
   being reached.
+* [`litcli`: global flags to allow overrides via environment variables](https://github.com/lightninglabs/lightning-terminal/pull/1007) `lncli` --help for more details.
```

Comment:
I would potentially put this under the `### Functional Changes/Additions` section instead. Also, feel free to add your name or github profile name under `# Contributors (Alphabetical Order)` if you'd like to be represented there.